public/index.html

```diff
@@ -30,9 +30,9 @@
     <script>
       window.pets = {
         informationAbout:
-          "If you're interested in adopting an animal, please have its Animal ID: {animalId} handy. Before calling or visiting our shelter, please review our page <a href='https://beta.baltimorecountymd.gov/departments/animalservices/how-to-adopt.html'>how to adopt</a>.",
+          `<h3>Information About Working Cat Adoption</h3><p>Adopt one or several of our working cats shown below and help save lives today! Learn more about our<a href='https://www.baltimorecountymd.gov/departments/animalservices/working-cat-program.html'>Working Cat Program</a>.</p>`,
```

Comment:
We need a space between "Learn more about our" and the link 😄

Reply:
Hahaha ugh, can you make that change for me?

Reply:
done!